src/libmongoc/tests/json/change_streams/unified/change-streams.json

```diff
@@ -5,8 +5,7 @@
     {
       "minServerVersion": "3.6",
       "topologies": [
```

Comment:
CDRIVER-3973 includes a new specification test: [expectedEventsForClient-ignoreExtraEvents.json](https://github.com/mongodb/specifications/commit/ff0c705f9cc95a5a9c8c595dbf9384cb41dfcdd2#diff-d301f7756cded2fb89e683f6fe148a65421513458837447bdbdf883c1c9e53ca). Please copy that in to `./src/libmongoc/tests/json/unified/`. It passes locally with this change set.